src/components/button/button.tsx

```diff
@@ -75,6 +74,14 @@ export class Button
   /**  When `true`, interaction is prevented and the component is displayed with lower opacity. */
   @Prop({ reflect: true }) disabled = false;
 
+  /**
+   * The ID of the form owner to associate this component with.
+   *
+   * When not set, this component will be associated with its ancestor `<form>` element, if any.
```

Comment:
Nitpick: we have been saying "the component" rather than "this component" in all of the API ref.  Applies to the other components as well.